src/__tmpdata/navigation.json

```diff
@@ -0,0 +1,194 @@
+{
+  "themes": [
```

Comment:
Would it make sense to structure this more like a graphql response? That might mean less changes once we put this in the CMS. The `allTopics` response is probably the best starting point for a structure like that.

```
{
  "data": {
    "allThemes": {
      "edges": [
        {
          "node": {
            "title": "womp womp",
            "slug": "womp-womp"
```

etc